lib/models/store_transaction.freezed.dart

```diff
@@ -193,25 +193,23 @@ class _$_StoreTransaction implements _StoreTransaction {
 
 abstract class _StoreTransaction implements StoreTransaction {
   const factory _StoreTransaction(
-      @JsonKey(name: 'revenueCatIdentifier')
-          final String revenueCatIdentifier,
-      @JsonKey(name: 'productIdentifier')
-          final String productIdentifier,
-      @JsonKey(name: 'purchaseDate')
-          final String purchaseDate) = _$_StoreTransaction;
+          @JsonKey(name: 'revenueCatId') final String revenueCatIdentifier,
+          @JsonKey(name: 'productId') final String productIdentifier,
+          @JsonKey(name: 'purchaseDate') final String purchaseDate) =
+      _$_StoreTransaction;
```

Comment:
Small thing but I'm curious. Are we setting a 80 char line length in flutter/dart? 

I think that's the standard, but personally I'm a fan of 120 characters. That said, not going to change it unless we want to

Reply:
Good question, we are following the default lint rules for flutter, I guess we could change it because it's super short

Reply:
+1 for 120, it's the char limit for GitHub not to start scrolling horizontally, and usually it's still thin enough to fit most screens

Reply:
https://github.com/dart-lang/dart_style/issues/833 that's a long issue lol

Reply:
I looked a bit into this and I don't think there's a way of change the config for both `flutter format` and the linter 🤔 I think there are only ways of changing it in your IDE and a way to pass it as a parameter to the linter